### ISSUE TITLE
ScaleSlider doesn't have name `prop`

### ISSUE THREAD

**Issue by christopherGdynia** [telekom/scale#883]
Hello there,

the `ScaleSlider` `Component` doesn't have the `name` `prop`. Can you add it, for `form` `accessibility`.

Best Regards :D

Same Issue like #616

**Comment by nowseemee** [comment]
Hi @christopherGdynia, yes, totally - will add this soon and let you know. Thanks! :D

**Comment by nowseemee** [comment]
Hi @christopherGdynia - I've created a PR ☝🏻 . While at it, I realized that probably this addition won't help you since the slider is using Shadow DOM. Can you check the PR and share your thoughts? Thanks.

**Comment by christopherGdynia** [comment]
Hi @nowseemee, I will use this with `Formik` in `React` and I am building a wrapper component to make it work. ^^

Die PR sieht soweit gut aus, danke dir 👍

**Comment by nowseemee** [comment]
I see, awesome! I just published `3.0.0-beta.50` 👍🏻
